pkg/distro/BUILD

```diff
@@ -97,6 +97,7 @@ genrule(
 bzl_library(
     name = "rules_pkg_lib",
     srcs = [
+        "//:internal/util.bzl",
```

Comment:
So, instead of creating a `private` package, you're going for `internal`?

Nothing necessarily wrong with that; most rulesets are taking the `private` route.  I recall you saying you didn't like that as (IIRC) it can be a bit confusing to users.

Reply:
I don't recall one or the other. My quick check shows most are private. I'll change to that.